helm/chart-operator/values.yaml

```diff
@@ -1,11 +1,20 @@
 clusterDNSIP: 172.31.0.10
 
 e2e: false
-cnr:
-  address: https://quay.io
 
 externalDNSIP: 8.8.8.8
 
+cluster:
+  kubernetes:
+    domain: cluster.local
+
+deployment:
+  requests:
+    cpu: 50m
+    memory: 100Mi
+  limits:
+    memory: 250Mi
```

Comment:
This is the previous limit that was removed in https://github.com/giantswarm/chart-operator/pull/592

chart-operator OOMed in a couple of control planes but its likely this was due to the too many secrets problem.